nrf_security/doc/nrf_security.rst

```diff
@@ -81,6 +82,20 @@ To enable the Arm CryptoCell CC310 backend, set the :option:`CONFIG_CC310_BACKEN
 .. note:: This backend is only available in nRF52840 and nRF9160.
 
 
+nrf_oberon backend
+------------------
+
+The :ref:`nrf_oberon_readme` library is a binarly library that provides select cryptographic algorithms optimized for use in nRF SOC's.
+This provides faster execution than standard mbed TLS implementations.
+
+The nrf_oberon backend provides support for AES ciphers, SHA-1 and SHA-256, ECC (ECDH, ECDSA, and ECJPAKE) using secp256r1.
+
+Enabling the nrf_oberon backend
+-------------------------------
+
+To use the :ref:`nrf_oberon_readme` as a backend, set the :option:`CONFIG_OBERON_BACKEND` Kconfig variable to true.
```

Comment:
this option is not found and cannot be linked when building, is it correct?

Reply:
This should hopefully been fixed now that we have the actual library in